src/fdk.ts

```diff
@@ -48,6 +48,38 @@ async function checkTokenExpired(auth_token) {
     }
 }
 
+const extraSentryDetails = () => {
+    const auth_token = configStore.get(CONFIG_KEYS.AUTH_TOKEN);
+    const organization = configStore.get(CONFIG_KEYS.ORGANIZATION);
+    const user = {};
+    if (auth_token?.current_user) {
+        const activeEmail = auth_token.current_user?.emails?.find?.((e) => e.active && e.primary)?.email ?? 'Not primary email set';
+        const name = `${auth_token.current_user?.first_name} ${auth_token.current_user?.last_name}`;
+        user['name'] = name;
+        user['email'] = activeEmail;
+    }
+    return {
+        command: `fdk ${process?.argv?.slice?.(2)?.join(" ")}`,
+        env: configStore.get(CONFIG_KEYS.CURRENT_ENV_VALUE),
+        user,
+        organization,
+    }
+}
+
+// catch unhandled error
+process.on('uncaughtException', (err: any) => {
```

Comment:
I think unhandled exception will be by default handled by sentry so not required.

Reply:
In few sentry errors, we were getting uncaught error because of 403. So if user is not Authorised and if we are getting uncaught error, we shouldn't send sentry.